Beocreate2/beo-extensions/ui-settings/menu.html

```diff
@@ -65,6 +65,15 @@ <h2>Connected display</h2>
 					<div class="menu-label">Use Connected Display</div>
 					<div class="menu-toggle"></div>
 				</div>
+
+				<div class="menu-label">Screensaver Timeout</div>
+
+				<div class="slider-wrap">
+					<div class="symbol" style="-webkit-mask-image: url(extensions/ui-settings/symbols-black/clock.svg); mask-image: url(extensions/ui-settings/symbols-black/clock.svg);"></div>
+					<div class="screensaver-timeout-slider black show-value"></div>
+					<div class="symbol" style="-webkit-mask-image: url(extensions/ui-settings/symbols-black/clock.svg); mask-image: url(extensions/ui-settings/symbols-black/clock.svg);"></div>
+				</div>
+
```

Comment:
I'm not sure this slider should be hidden when the "Use Connected Display" is hidden, but for now it is.